src/server/list_family.cc

```diff
@@ -853,6 +891,77 @@ OpResult<string> ListFamily::OpIndex(const OpArgs& op_args, std::string_view key
   return str;
 }
 
+OpResult<vector<uint32_t>> ListFamily::OpIndex(const OpArgs& op_args, std::string_view key,
+                                               std::string_view element, ArgSlice args_ext) {
```

Comment:
We already have `OpIndex` that returns a value at index. This does something different, so lets maybe name it differently for less confusion? Like `OpPos` or something similar